modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,148 @@
+= Policies Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+//where are we covering what a policy? What the policy does
+
+Policies differ based on several different factors, such as level of granularity,types of policies, and policy categories. You must carefully review these factors, before you decide to implementation them in your environment. In this section, each of these factors are explained in detail. 
+
+// == Policy Components
+
+// A policy consists of the following components:
+
+// * YAML file
+// +
+// Defines the configurable parameters of the policy. Rendered by the UI to display the input of the policy.
+// +
+// * XML template
+// +
+// Contains the implementation of the policy.
+// * Template's POM
+// // Is it clear in the navigation that this is only for Mule 4? Mule 3 policies don't have a POM or a descriptor. -- Eva?
+// // What is POM?
+// +
+// Defines policy dependencies. Packaging type needs to be `mule-policy`.
+// * Resources
+// +
+// Optionally if the policy depends on other resources, such as a certificate, define resources as part of the policy.
+// +
+// * Mule-artifact.json descriptor
+// +
+// Specify the classifier as `mule-policy`.
+
+// For more information, see the header policy.
+
+== Policy Scope
+
+Policies are isolated from the application and other policies. However, there are ways to expose information from a policy. The authentication information about a policy can be propagated in the Security Context Principal object.
+
```

Comment:
I would add: 

"Some policies, like the token enforcement policy allow propagation of headers including the information related to the token to be used by the implementation service or by the API itself"

Reply:
Done! Thanks!